WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersViewModel.kt

```diff
@@ -92,7 +96,10 @@ class BloggingRemindersViewModel @Inject constructor(
     private fun buildPrologue() = listOf(
             Illustration(R.drawable.img_illustration_celebration_150dp),
             Title(UiStringRes(R.string.set_your_blogging_goals_title)),
-            HighEmphasisText(UiStringRes(R.string.set_your_blogging_goals_message)),
+            if (!isFromSiteSettings) {
+                HighEmphasisText(UiStringRes(R.string.set_your_blogging_goals_message))
+            } else {
+                HighEmphasisText(UiStringText("")) },
```

Comment:
I think we should just not add the item to the list in this case

Reply:
I've refactored to use your builder after merging